doc/modules/ROOT/pages/usage/tls.adoc

```diff
@@ -0,0 +1,56 @@
+= TLS (Transport Layer Security) support
+
+The standard way to securely access a nREPL server has been
+to use `ssh` with port forwarding, and letting the nREPL server only bind to `127.0.0.1`.
+
+Sometimes however `ssh` may not be available, and thus you need to
+bind to `0.0.0.0` to make the nREPL server available. The following will explain
+how to do that with TLS.
+
+== Generating certificates and keys
+
+[source,shell]
+----
+$ clojure -Sdeps '{:aliases {:write-cert {:deps {com.github.ivarref/locksmith {:mvn/version "0.1.6"}} :exec-fn com.github.ivarref.locksmith/write-certs!}}}' -T:write-cert :duration-days 365
```

Comment:
Might be good to also show some examples how to generate the certs with standard Unix utils (if possible).